apis/python/tests/test_dataframe.py

```diff
@@ -255,6 +255,127 @@ def test_dataframe_with_enumeration(tmp_path):
         assert_array_equal(df["myfloat"].chunk(0).dictionary, enums["enmr2"])
 
 
+@pytest.mark.parametrize("mode", ["r", "w"])
```

Comment:
Missing test:  ensuring timestamps/time travel and enum changes work correctly, i.e., that you can open at a previous time and see the correct values returned by get_enum_values

Reply:
Pushed